src/export.js

```diff
@@ -1,5 +1,6 @@
 // For browser, export only select globals
-if ( typeof window !== "undefined" ) {
+if ( window.document && window.HTMLDocument &&
+		window.document instanceof window.HTMLDocument ) {
```

Comment:
This seems like the wrong approach. We shouldn't alias `global` to `window` in the first place.